pom.xml

```diff
@@ -261,7 +261,7 @@
             </goals>
             <configuration>
               <failOnViolation>true</failOnViolation>
-              <excludes>org/apache/zeppelin/interpreter/thrift/*</excludes>
+              <excludes>org/apache/zeppelin/interpreter/thrift/*,org/apache/zeppelin/scio/avro/*,org/apache/zeppelin/scio/avro/*</excludes>
```

Comment:
@khalidhuseynov OMG! It's my mistake. It works well with only one path. But this PR is already merged. Should I open new PR of which title begins with [HOTFIX]? Thanks for pointing this out :)